ReplayKitExample/BroadcastExtension/SampleHandler.swift

```diff
@@ -0,0 +1,227 @@
+//
+//  SampleHandler.swift
+//  BroadcastExtension
+//
+//  Created by Piyush Tank on 7/1/18.
+//  Copyright © 2018 Twilio. All rights reserved.
+//
+
+import Accelerate
+import TwilioVideo
+import ReplayKit
+
+class SampleHandler: RPBroadcastSampleHandler, TVIRoomDelegate, TVIVideoCapturer {
+
+    public var isScreencast: Bool = true
+
+    // Video SDK components
+    public var room: TVIRoom?
+    weak var captureConsumer: TVIVideoCaptureConsumer?
+
+    static let kDesiredFrameRate = 30
+    static let kDownScaledFrameWidth = 540
+    static let kDownScaledFrameHeight = 960
```

Comment:
Make the scaling logic aspect ratio independent so it can work on iPad, iPhone, iPhone X. Consider setting a maximum dimension (either width or height) of 640.